site/docs/samples/notebook-read.md

```diff
@@ -72,44 +72,106 @@ stringData:
 EOF
 ```
 
-Then, register the data asset itself in the data catalog `katalog` used for samples. Replace the values for `endpoint`, `bucket` and `object_key` with values from the object storage service that you used and run:
+Next, register the data asset itself in the data catalog. The way to do it depends on whether you use katalog or OpenMetadata as your data catalog:
 
-```yaml
-cat << EOF | kubectl apply -f -
-apiVersion: katalog.fybrik.io/v1alpha1
-kind: Asset
-metadata:
-  name: paysim-csv
-spec:
-  secretRef: 
-    name: paysim-csv
-  details:
-    dataFormat: csv
-    connection:
-      name: s3
-      s3:
-        endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
-        bucket: "demo"
-        object_key: "PS_20174392719_1491204439457_log.csv"
-  metadata:
-    name: Synthetic Financial Datasets For Fraud Detection
-    geography: theshire 
-    tags:
-      finance: true
-    columns:
-      - name: nameOrig
-        tags:
-          PII: true
-      - name: oldbalanceOrg
-        tags:
-          PII: true
-      - name: newbalanceOrig
+=== "katalog"
+    
+    Replace the values for `endpoint`, `bucket` and `object_key` with values from the object storage service that you used and run:
+    
+    ```yaml
+    cat << EOF | kubectl apply -f -
+    apiVersion: katalog.fybrik.io/v1alpha1
+    kind: Asset
+    metadata:
+      name: paysim-csv
+    spec:
+      secretRef:
+        name: paysim-csv
+      details:
+        dataFormat: csv
+        connection:
+          name: s3
+          s3:
+            endpoint: "http://localstack.fybrik-notebook-sample.svc.cluster.local:4566"
+            bucket: "demo"
+            object_key: "PS_20174392719_1491204439457_log.csv"
+      metadata:
+        name: Synthetic Financial Datasets For Fraud Detection
+        geography: theshire
         tags:
-          PII: true
-EOF
-```
-
-The asset is now registered in the catalog. The identifier of the asset is `fybrik-notebook-sample/paysim-csv` (i.e. `<namespace>/<name>`). You will use that name in the `FybrikApplication` later.
+          finance: true
+        columns:
+          - name: nameOrig
+            tags:
+              PII: true
+          - name: oldbalanceOrg
+            tags:
+              PII: true
+          - name: newbalanceOrig
+            tags:
+              PII: true
+    EOF
+    ```
+    
+    The asset is now registered in the catalog. The identifier of the asset is `fybrik-notebook-sample/paysim-csv` (i.e. `<namespace>/<name>`). You will use that name in the `FybrikApplication` later.
+
+=== "OpenMetadata"
+
+    We use port-forwarding to send asset creation requests to the OpenMetadata connector.
+    ```bash
+    kubectl port-forward svc/openmetadata-nnector -n fybrik-system 8081:8080 &
+    cat << EOF | curl -X POST localhost:8081/createAsset -d @-
+    {
+      "destinationCatalogID": "openmetadata",
```

Comment:
Does OpenMetadata have sub-catalogs, such as do other proprietary solutions?  I.e. Can different values be provided here, or does it always have to be "openmetadata"?

Reply:
OpenMetadata does not have sub-catalogs.
The `destinationCatalogID` is used in constructing the default name for the OM `databaseService`.
For instance, if the `destinationCatalogID` is `openmetadata` and the connection type is `s3`, then the connector will try to create a `databaseService` called `openmetadata-s3` (and it will succeed, unless this name is already taken).